### ISSUE TITLE
[BUG] Wrong temperature for global CPU on FreeBSD 13.0

### ISSUE THREAD

**Issue by lcheylus** [aristocratos/btop#295]
I have a bug on FreeBSD 13.0/amd64 with the last version of btop (commit d14af7f0e0510c2a611f03d5ce2500042e5dba53).

With a Atom 330 CPU (4 threads), the temperature for global CPU is always -273°C ! The temperatures for the 4 CPU (thread) are correct.

```
$ sysctl dev.cpu|grep temperature
dev.cpu.3.temperature: 47.0C
dev.cpu.2.temperature: 47.0C
dev.cpu.1.temperature: 42.0C
dev.cpu.0.temperature: 43.0C
```
**Screenshots**

![btop-FreeBSD13](https://user-images.githubusercontent.com/3294998/159973636-7aa95997-b873-457c-9dae-b915e34b8066.png)

**Screenfetch** : 

OS: FreeBSD
 Kernel: amd64 FreeBSD 13.0-RELEASE-p4
 Uptime: 168d 8h 25m
 Packages: 145
 Shell: bash (bash)
 Disk: 2.1T / 8.2T (26%)
 CPU: Intel Atom 330 @ 1.60GHz
 GPU: 82945G/GZ Integrated Graphics Controller
 RAM: 777MiB / 2048MiB

 - btop++ version: 1.2.5
 - Binary: self compiled on FreeBSD 13.0/amd64
 - Compiler and version: gcc version 11.2.0 (FreeBSD Ports Collection)
 - Architecture: amd64
 - Platform: FreeBSD 13.0
 - Terminal used: Kitty https://sw.kovidgoyal.net/kitty/
 - Font used: Cousine Nerd Font Mono

**Comment by aristocratos** [comment]
@lcheylus 
What output do you get from `sysctl hw.acpi.thermal` ?

**Comment by lcheylus** [comment]
Hi @aristocratos 

no entry for `sysctl hw.acpi.thermal` : 

```
$ sysctl hw.acpi.thermal
sysctl: unknown oid 'hw.acpi.thermal'
```

**Comment by aristocratos** [comment]
@lcheylus 
That would explain it, it's currently looking for cpu package temp from `hw.acpi.thermal.tz0.temperature` and it seems there is no check if that value is valid.

Will look in to adding some code to calculate a mock package temp from the average of the core temps if this value is missing.

**Comment by aristocratos** [comment]
@lcheylus 
Try with latest commit.

**Comment by lcheylus** [comment]
@aristocratos 

OK, problem fixed on FreeBSD-13.0 with the last commit : global temp for CPU is correct.

Thanks.

**Comment by JoshuaPettus** [comment]
And they said absolute zero was impossible to reach...
